sumomongodbatlascollector/main.py

```diff
@@ -130,6 +130,11 @@ def _get_process_names(self):
 
         processes = self.kvstore.get('processes')
         process_ids, hostnames = processes['process_ids'], processes['hostnames']
+		process_ids_copy = process_ids.copy()
```

Comment:
why are we creating copies and deleting originals? also process_ids_copy is not used

Reply:
have removed the process_ids_copy and hostname_copy